configure.py

```diff
@@ -1559,10 +1594,9 @@ def generate(self):
             self._add_target_details(tests_target_configuration, module)
             self._add_target_details(cli_target_configuration, module)
 
-        library_target_libs_and_frameworks = '%s %s %s' % (
+        library_target_libs_and_frameworks = '%s %s' % (
             ' '.join(library_target_configuration['frameworks']),
-            ' '.join(library_target_configuration['libs']),
-            self._cc.mach_abi_link_flags(self._options)
+            ' '.join(library_target_configuration['libs'])
```

Comment:
Let's keep the optional trailing comma, reducing the diff here and for future changes to the list